mdoc-sbt/src/main/scala/mdoc/MdocPlugin.scala

```diff
@@ -66,6 +66,11 @@ object MdocPlugin extends AutoPlugin {
     }
   }
 
+  lazy val mdocJSWorkerClasspath = taskKey[Option[Seq[File]]](
+    "Optional classpath to use for Mdoc.js worker - " +
+      "if not provided, the classpath will be formed by resolving the worker dependency"
+  )
+
```

Comment:
We need this separate task (**not part of autoImport**) to make mdoc's own tests to work - we can't introduce a dependency on mdoc-js-worker, and this would be reliable than placing `dependsOn(jsWorker / publishLocal)` in all the possible situations where this module must be available.